product_taxes_group/model/tax_group.py

```diff
@@ -0,0 +1,213 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Product - Taxes Group module for Odoo
+#    Copyright (C) 2014 -Today GRAP (http://www.grap.coop)
+#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import logging
+
+from openerp.osv import fields, osv
+from openerp.osv.orm import Model
+from openerp import SUPERUSER_ID
+from openerp.tools.translate import _
+
+_logger = logging.getLogger(__name__)
+
+
+class tax_group(Model):
```

Comment:
Hi. It doesn't work. Functional function like _get_product_ids must be set before the _columns section. Otherwise, I have the following error : 

    File "~/account-financial-tools/product_taxes_group/model/tax_group.py", line 52, in tax_group
    _get_product_ids, type='one2many', relation='product.product',
    NameError: name '_get_product_ids' is not defined

Thanks for your review. can you put a comment :+1: if all is OK for you ? Thanks.
Regards.